configure

```diff
@@ -1083,6 +1085,50 @@ do
                 err "musl libc $CFG_MUSL_ROOT/lib/libc.a not found"
             fi
             ;;
+
+        x86_64-*-msvc)
+            step_msg "targeting MSVC, disabling jemalloc"
+            CFG_DISABLE_JEMALLOC=1
+            putvar CFG_DISABLE_JEMALLOC
+
+            probe_need CFG_CMAKE cmake
+            probe CFG_CL cl
+            if [ -z "$CFG_CL" ]; then
+                if [ -z "$CFG_MSVC_ROOT" ]; then
+                    err "msvc root not found, pass --msvc-root or ensure \
+                         a 64-bit cl.exe is in your PATH"
+                fi
+                CFG_CL="$CFG_MSVC_ROOT/VC/bin/amd64/cl.exe"
```

Comment:
For future reference, there is also a 32-bit cl.exe that can target 64-bit. You know, in case some weirdo really wants to cross compile from a 32-bit OS.